src/js/wrap.cpp

```diff
@@ -1725,7 +1757,8 @@ namespace ccf::js
 
     auto& tx = *tx_ptr;
 
-    js::Context ctx2(js::TxAccess::APP);
+    // TODO
```

Comment:
?

Reply:
Left a few TODOs while I was converting the old enums to the new. In this case (parsing modules to build bytecode) I _think_ it would be safe to have `RO`, or even no exec context at all - surely we do no execution here so the bindings don't matter? But haven't investigated a further refactor to support this, so just leaving as `RW` for now.